lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -20,38 +20,69 @@ function canFindDenominatorInNumerator(node) {
   if (Node.Type.isParenthesis(denominator)) {
     denominator = denominator.content;
   }
-  if (!(numerator.op === '+' || numerator.op === '-' ||
-        denominator.op === '+' || numerator.op === '-')) {
+
+  let n_args_length;
+  // If numerator is '*' op, it signifies a single 'ax', should be assigned a
+  // length of 1
+  if ('args' in numerator && numerator.op !== '*') {
+    n_args_length = numerator.args.length;
+  }
+  else {
+    n_args_length = 1;
+  }
+  let d_args_length;
+  if ('args' in denominator) {
+    d_args_length = denominator.args.length;
+  }
+  else {
+    d_args_length = 1;
+  }
+
+  // If numerator isn't length 2 or length 1 with a polynomial return false
+  if (!(n_args_length === 2)) {
+    if (!(n_args_length === 1 || Node.Type.isConstant(numerator))) {
```

Comment:
yeah that would be good to add there if it's easy! probably not instead, since it's nice to have the logic within polynomial term code, but if you could reference it in `Type` for now that'd be awesome